torch/fx/graph.py

```diff
@@ -952,6 +952,20 @@ def check_arg(arg : Node, n : Optional[Node] = None) -> None:
                             raise RuntimeError(f'Node {node} target {node.target} references nonexistent attribute '
                                                f'{atom} of {seen_qualname}')
 
+    def run_dce(self):
+        """
+        Remove all dead code from the graph, based on each node's number of users.
```

Comment:
Can you format this according to the Google Python docstring style: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Would probably also be nice to have a quick example in the docstring